pkg/bitcoin/electrum/config.go

```diff
@@ -0,0 +1,13 @@
+package electrum
+
+// BitcoinConfig defines the configuration for Bitcoin.
+type BitcoinConfig struct {
```

Comment:
What about naming it just `Config`? We would have `electrum.Config` living in `pkg/bitcoin/electrum` referenced from `[bitcoin.electrum]` section. I think it all makes sense.

Reply:
There is already such a [structure](https://github.com/keep-network/keep-core/blob/c87ccffd4c948c25b629641e022c91451826225a/pkg/bitcoin/electrum/config.go#L10)
 in the electrum package and it contains config for electrum (it was added by Kuba and I copied it). It's the structure used by BitcoinConfig.

Reply:
Right, I think we should have just one config (cc @nkuba) structure named `Config`:
```
package electrum

// Config defines the configuration for the Electrum Bitcoin client.
type Config struct {
  // URL to the Electrum server in format: `hostname:port`.
  URL string
}
```

Based on [this](https://github.com/keep-network/keep-core/pull/3374#discussion_r1024157900) conversation, we agreed that we are *not* configuring Bitcoin but we are configuring Electrum Bitcoin client. The configuration is in `[bitcon.electrum]` in the config and the structure is in `pkg/bitcoin/electrum/config.go`.

Reply:
If we want to use a nested structure in the config file:
```
[bitcoin.electrum]
URL = "url.to.electrum:1111"
```
then the corresponding structure in Golang should also be nested:
```
type BitcoinConfig struct {
 Electrum Config
}

type Config struct {
 	URL string
}
```.

If we don't use a nested structure, but just a simple non-nested `Config` structure, then the `Viper` package that we use for mapping config file to the Golang structure will be unable to read the configuration properly.

Reply:
I proposed an update in https://github.com/keep-network/keep-core/pull/3406/commits/40ff528ac1797d6ce3cf2a449ef745e10480b538